heron/scheduler-core/tests/java/com/twitter/heron/scheduler/SchedulerMainTest.java

```diff
@@ -69,42 +71,34 @@ public void setUp() throws Exception {
         when(config.getStringValue(ConfigKeys.get("STATE_MANAGER_CLASS"))).
         thenReturn(STATE_MANAGER_CLASS);
     Mockito.
-        when(config.getStringValue(ConfigKeys.get("PACKING_CLASS"))).
-        thenReturn(PACKING_CLASS);
-    Mockito.
         when(config.getStringValue(ConfigKeys.get("SCHEDULER_CLASS"))).
         thenReturn(SCHEDULER_CLASS);
 
-    String iTopologyName = "topologyName";
     int iSchedulerServerPort = 0;
 
     TopologyAPI.Topology topology =
         TopologyTests.createTopology(
             iTopologyName, new com.twitter.heron.api.Config(),
             new HashMap<String, Integer>(), new HashMap<String, Integer>());
-    String packingString = "dummyPackingString";
 
     // Mock objects to be verified
     stateManager = Mockito.mock(IStateManager.class);
-    packing = Mockito.mock(IPacking.class);
     scheduler = Mockito.mock(IScheduler.class);
 
-    PackingPlan packingPlan = Mockito.mock(PackingPlan.class);
-    Mockito.when(packing.pack()).thenReturn(packingPlan);
-    Mockito.when(packingPlan.getInstanceDistribution()).thenReturn(packingString);
-
+    Map<String, PackingPlan.InstancePlan> instances = new HashMap<>();
+    instances.put("1:1:1:1",
+        new PackingPlan.InstancePlan("1:1:1:1", "dummy", new Resource(1, 1, 1)));
     Map<String, PackingPlan.ContainerPlan> containers = new HashMap<>();
-    containers.put("dummy", new PackingPlan.ContainerPlan("dummy", null,
-        new Resource(1, 1, 1)));
-
-    Mockito.when(packingPlan.getContainers()).thenReturn(containers);
+    containers.put("1", new PackingPlan.ContainerPlan("1", instances, new Resource(1, 1, 1)));
+    PackingPlan packingPlan = new PackingPlan("packing-id", containers, new Resource(1, 1, 1));
+    final SettableFuture<PackingPlans.PackingPlan> future = SettableFuture.create();
+    future.set(new PackingPlanProtoSerializer().toProto(packingPlan));
```

Comment:
I've included the change in #1321, which I suspect could merge this today or tomorrow. It's up to you, you could either wait and refactor this PR or do it as a followup.